packages/react-components/react-tags/src/components/Tag/TagContext.ts

```diff
@@ -0,0 +1,18 @@
+import { createContext, ContextSelector, useContextSelector } from '@fluentui/react-context-selector';
+import type { Context } from '@fluentui/react-context-selector';
+import type { TagContextValue } from './Tag.types';
+
+export const TagContext: Context<TagContextValue> = createContext<TagContextValue | undefined>(
+  undefined,
+) as Context<TagContextValue>;
+
+const tagContextDefaultValue: TagContextValue = {
+  dismissible: false,
+  shape: 'rounded',
+  size: 'medium',
+  interactive: false,
+};
+
+export const TagProvider = TagContext.Provider;
+export const useTagContext_unstable = <T>(selector: ContextSelector<TagContextValue, T>): T =>
```

Comment:
do you really need context selectors for this? These values are all primitive and are not likely to change often like with an `open` prop.

You could keep this is a normal context and memoize the value, there's nothing wrong consuming the entire object from context and destructuring the result

Reply:
Yeah TBH not really. I'm consuming the entire context actually and not just part of it. I was going back and force there thinking it could be nice if more stuff is added to the context. But Let me remove the context selector for now.

Reply:
context selection removed :)